src/bobril.ts

```diff
@@ -1542,6 +1547,8 @@ b = ((window: Window, document: Document): IBobrilStatic => {
         cloneNode: cloneNode,
         shimStyle: shimStyle,
         flatten: flatten,
-        mergeComponents: mergeComponents
+        syncUpdate: syncUpdate,
+        mergeComponents: mergeComponents,
+        
```

Comment:
Be more careful about formatting there was no reason to add comma and empty line...